server/src/domain/asset/asset.service.ts

```diff
@@ -420,11 +427,54 @@ export class AssetService {
       await this.assetRepository.updateAll([options.stackParentId], { stackParentId: null });
     }
 
+    if (creationDate !== undefined || latitude !== undefined || longitude !== undefined) {
+      this.validateGpsCoordinates(latitude, longitude);
+      for (const id of ids) {
+        await this.updateExifData(id, { CreationDate: creationDate, GPSLatitude: latitude, GPSLongitude: longitude });
```

Comment:
It's unclear to me if we want to write this directly, or if we want to queue a job that will refresh the data in the background. Especially for the bulk update, I think a job would make more sense. We can use a web socket event to notify the client when it is done.

Reply:
Yeah I think it certainly better fit inside a job, I will migrate this inside

Reply:
In that case we can do something like `if(metadata update requested) { await this.save(updates); await this.jobRepo.queue({ name: JobName.WRITE_EXIF, data: { id: asset.id }}) }`

Reply:
Migration done